### PR TITLE
Add logging structure to documentation

### PR DESCRIPTION
**Pull request by amoriartyCH**
Create a logging document inside of the documentation directory of our service. This document will display notable log lines and variables which are of interest to other teams, front end support and logging tools such as Kibana.

**Resolves DSND-171**

### CODE REVIEW

**Review by ch-tstrickland — COMMENTED** [review]
I wonder if context-id could be pulled into a separate section. That section could also mention the context-id's use in Kafka, pointing to the Avro field used. It could also point out where the value comes from (ERIC X-Request-ID) that this is the only means of tracking messages through from CHIPS and so through the other services. TODO: you need a meeting with Thundercats to ensure that they use this context-id in their logging and that they specify it as the context id in their calls to other services.

File: docs/logging.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+`"offset":offset_int, "partition":partition_int` - These values are only returned if a message has successfully been sent to Kafka.
+
+With an example of this being:
+```go
+{"context":"zWJcenQdHgzRmoK1hL_ridjiybm1","created":"2021-09-13T12:19:29.825429905+01:00","data":{"message":"Sent message","offset":1,"partition":6,"topic":"officers-delta"},"event":"info","namespace":"chs-delta-api"}
+```
+---
+
+#### Message fails to be sent to Kafka
```
Comment (ch-tstrickland) [inline docs/logging.md:59]: Could you make the point that the messages vary quite a bit. Perhaps give the two examples shared by yourself & Faisal?

**Review by ch-tstrickland — COMMENTED** [review]
Generally this looks brilliant. I have made some very minor points, but if I were coming to this from support or similar I would be very happy.

File: docs/logging.md (hunk 1 above)
Comment (amoriartyCH) [reply docs/logging.md:59]: Both your first comment and this one have been implemented. Thanks for the review :)

**Review by ch-tstrickland — APPROVED** [review]

**Pull request merged**